packages/calcite-components/support/generateT9nDocsJSON.ts

```diff
@@ -0,0 +1,52 @@
+// generates a JSON file containing the per component t9n translation values
+(async () => {
+  const { dirname, resolve } = await import("path");
+  const { fileURLToPath } = await import("url");
+  const {
+    existsSync,
+    promises: { readFile, readdir, writeFile }
+  } = await import("fs");
+  try {
+    const __dirname = dirname(fileURLToPath(import.meta.url));
+
+    const outfile = resolve(__dirname, "..", "dist", "extras", "translations-json.json");
+    const assetsPaths = resolve(__dirname, "..", "dist", "calcite", "assets");
+    const components = await readdir(assetsPaths);
+
+    const data = {};
+    const messagesFilenameRegex = /messages_(.*)\.json/;
+
+    for (const component of components) {
+      const t9nPath = resolve(assetsPaths, component, "t9n");
+      if (existsSync(t9nPath)) {
+        data[component] = {};
+        // the english file is the only one without the lang in the filename
```

Comment:
English does have a corresponding bundle (see https://github.com/Esri/calcite-components/blob/master/packages/calcite-components/src/components/action/assets/action/t9n/messages_en.json). The one without a language code is the main one.

Could you update the regex pattern to match bundles with or without a language code?

Reply:
Oh gotcha. What should I be doing with the main one then, does it need to be included?  And if so what should I call the key, "main"? cc @macandcheese

Reply:
It seems like `messages.json` and `messages_en.json` are always the same, but I created a separate field called `main` for `messages.json` anyway.

Reply:
Yeah, in most cases they will be the same. WDYT about omitting `main` for now until we have a strong reason to include it? Sorry, this would mean reverting the regex update I suggested earlier. 😅